src/api/services/iuf/iuf.go

```diff
@@ -73,22 +74,29 @@ type IufService interface {
 
 // IufService service layer
 type iufService struct {
-	logger           utils.Logger
-	workflowCient    workflow.WorkflowServiceClient
-	k8sRestClientSet kubernetes.Interface
-	keycloakService  services_shared.KeycloakService
-	env              utils.Env
+	logger                 utils.Logger
+	workflowClient         workflow.WorkflowServiceClient
```

Comment:
Nice fix :)

Would you be willing to make this same spelling fix throughout the code? I notice another instance of it in the `workflowService` struct as well for both the `workflowCient` and `workflowTemplateCient` fields.

Reply:
Changed all variables in entire codebase from `workflowCient` to `workflowClient` and from `workflowTemplateCient` to `workflowTemplateClient`